### PR TITLE
Extract FencesBlockToken's message generator out

### PR DESCRIPTION
**Pull request by hellosnow**
Extract FencesBlockToken's message generator out:
1. Internal method to public, to be reused by nuget.
2. Separate with other helper method, incase the dfm renderer helper will get complex in the future

@qinezh @superyyrrzz @chenkennt @ansyral @vwxyzh

### CODE REVIEW

**Comment by hellosnow** [comment]
@vicancy

File: src/Microsoft.DocAsCode.Dfm/DfmBlockquoteHelper.cs [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Dfm
+{
+    using System.Collections.Generic;
+    using System.Collections.Immutable;
+
+    using Microsoft.DocAsCode.MarkdownLite;
+
+    internal class DfmBlockquoteHelper
```
Comment (hellosnow) [inline src/Microsoft.DocAsCode.Dfm/DfmBlockquoteHelper.cs]: test comment

**Comment by superyyrrzz** [comment]
:shipit:

File: src/Microsoft.DocAsCode.Dfm/DfmBlockquoteHelper.cs [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    {
+        public class SplitToken
```
Comment (vwxyzh) [inline src/Microsoft.DocAsCode.Dfm/DfmBlockquoteHelper.cs]: do not use public nested class.

**Comment by qinezh** [comment]
:shipit:

**Comment by vwxyzh** [comment]
🚀

File: src/Microsoft.DocAsCode.Dfm/DfmBlockquoteHelper.cs [hunk 3]
```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Dfm
+{
+    using System.Collections.Generic;
+    using System.Collections.Immutable;
+
+    using Microsoft.DocAsCode.MarkdownLite;
```
Comment (hellosnow) [inline src/Microsoft.DocAsCode.Dfm/DfmBlockquoteHelper.cs]: test

**Pull request merged**